libs/remix-ui/run-tab/src/lib/actions/events.ts

```diff
@@ -21,6 +23,8 @@ export const setupEvents = (plugin: RunTab, dispatch: React.Dispatch<any>) => {
   })
 
   plugin.blockchain.event.register('contextChanged', (context, silent) => {
+    dispatch(resetProxyDeployments())
+    if (context !== 'vm') getNetworkProxyAddresses(plugin, dispatch)
```

Comment:
with my custom fork PR, context cannot be "vm". you 'll have to use `context.startsWith('vm)'`

Reply:
Ok, cool.